src/utils.c

```diff
@@ -488,3 +488,26 @@ static int android_close(void *cookie)
     return 0;
 }
 #endif  // PLATFORM_ANDROID
+
+// From https://stackoverflow.com/questions/1659440/32-bit-to-16-bit-floating-point-conversion/60047308#60047308
+
+unsigned int AsUnsignedInt(const float x) {
+    return *(unsigned int*)&x;
+}
+float AsFloat(const unsigned int x) {
```

Comment:
Considering this function is only used 1-2 times and it's really minimal, I'd prefer to implement it directly in the required function.